packages/rancher-sriov/generated-changes/patch/Chart.yaml.patch

```diff
@@ -5,7 +5,7 @@
 +  catalog.cattle.io/auto-install: sriov-crd=match
 +  catalog.cattle.io/certified: rancher
 +  catalog.cattle.io/experimental: "true"
-+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.27.0-0'
++  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.29.0-0'
```

Comment:
Confirming if this is not a mistake. We want to support 1.28 as well ?

Reply:
nope, I got confused on the support matrix, it's fixed